Sources/Core/ModernAVPlayer.swift

```diff
@@ -110,6 +111,16 @@ public final class ModernAVPlayer: NSObject, MediaPlayer {
         }
         context.loadMedia(media: media, autostart: autostart)
     }
+   
+    /**
+     Replaces the current player metadata with the new metadata
+     This function is dedicated for updating live data, and though, will discard duration and isLive previously set.
```

Comment:
This method is dedicated for updating nowPlayingInfo data...

Reply:
ok